src/common/AndroidEnvReqResolver.ts

```diff
@@ -48,13 +51,45 @@ export class AndroidCommandLineToolsResolver implements RequirementResolver {
         }
     }
 
-    private installAndroidCommandLineTools(): Promise<string | undefined> {
-        process.env.ANDROID_HOME = '/Users/pvandyk/Library/Android/sdk';
+    private async installAndroidCommandLineTools(): Promise<
+        string | undefined
+    > {
+        // download command line tools
 
-        if (process.env.ANDROID_HOME) {
-            return Promise.resolve(this.resolvedMessage);
-        } else {
-            return Promise.reject(new SfdxError(this.unableToResolveMessage));
-        }
+        // unzip and move to the latest folder
+        const unzipCommand = ''.concat(
+            'unzip /Users/pvandyk/Downloads/commandlinetools-mac-8092744_latest.zip -d $ANDROID_HOME/cmdline-tools/latest',
+            ' && ',
+            'mv $ANDROID_HOME/cmdline-tools/latest/cmdline-tools/* $ANDROID_HOME/cmdline-tools/latest',
+            ' && ',
+            'rm -rf $ANDROID_HOME/cmdline-tools/latest/cmdline-tools'
+        );
+        CommonUtils.executeCommandSync(unzipCommand);
+
+        // this is needed because unzipping doesn't register the sdkmanager
+        // it will install to latest-2, so remove and install
+        const installCommand = ''.concat(
```

Comment:
@pvandyk changes that go into this repo should work on mac/unix/linux and on windows platforms (take a look [here](https://github.com/forcedotcom/lwc-dev-mobile-core/blob/6edd4c22a22517d52f46616ff49a857978a05e6d/src/common/AndroidUtils.ts#L840-L842) for an example). I don't think these commands work on Windows as they are right now.